dll/win32/shell32/dialogs/view.cpp

```diff
@@ -249,7 +249,7 @@ ViewTree_InsertAll(HWND hwndTreeView)
 static BOOL
 ViewTree_LoadTree(HKEY hKey, LPCWSTR pszKeyName, DWORD dwParentID)
 {
-    DWORD dwIndex;
+    DWORD dwIndex = ~0UL;
```

Comment:
At line 293 this: `pEntry->dwResourceID = DWORD(-1);`  was done; could you use `DWORD(-1)` for initializing dwIndex there as well?